src/types/Todo.ts

```diff
@@ -1,4 +1,4 @@
-export interface Todo {
+export interface typeTodo {
```

Comment:
```suggestion
export interface TypeTodo {
```
Check the linter error (do not ignore them - if you have questions it's better to ask in the chat) - types should be named using PascalCase